e2e/Makefile

```diff
@@ -3,54 +3,54 @@ TEST_KUBERNETES_TARGET ?= current
 TEST_SCHEDULER_MANIFEST ?= daemonset
 
 ## Dependency versions
-KIND_VERSION=0.10.0
-MINIKUBE_VERSION=v1.18.1
+KIND_VERSION := 0.10.0
+MINIKUBE_VERSION := v1.18.1
 
-BINDIR := $(shell pwd)/bin
-SUDO=sudo
-KIND_CLUSTER_NAME=topolvm-e2e
+BINDIR := $(PWD)/bin
```

Comment:
keep using `$(shell pwd)`.  It's intentional.
https://jerome-wang.github.io/2015/08/13/pwd-in-sudo-make/